### ISSUE TITLE
Show in advance the limit of message in private chats

### ISSUE THREAD

**Issue by victor-ponamariov** [bluesky-social/social-app#6246]
### Describe the Feature

When you write a long message to someone, then hit enter, you got this toast at the left corner.

It would be much better to have some kind of indicator of how many characters are left. The design can be different, there are many ways to display it. 

By the way, 1 431 chars are not that big in my opinion (that's was my message length). In any case, now I need to guess what the limit is and split the message.

![CleanShot 2024-11-12 at 03 25 08@2x](https://github.com/user-attachments/assets/28bcad9a-5ae0-4e4f-b3a3-8df8cd8f44f5)

### Describe Alternatives

Just a couple of screenshots from google pictures. 

![image](https://github.com/user-attachments/assets/25d713a2-55a0-49b9-9ef8-94cbf295e18d)
![image](https://github.com/user-attachments/assets/5bb61f0e-0481-429b-9575-66ffc1c61eb4)
![image](https://github.com/user-attachments/assets/e3f710a1-e325-4dcf-bf16-9ee8b35fc692)

**Comment by wojtek-be** [comment]
Yes, please. I'm constantly hitting the limit recently and at the very least having a counter (current/max) would help A LOT!
